bin/ares-device.js

```diff
@@ -161,12 +168,16 @@ function getResourceMonitor() {
     }
     log.info("getResourceMonitor()", "interval:", options.interval);
 
-    if (argv.list) {
-        // print all running app and service's resource usage
+    if (argv["id-filter"]) {
+        // Handle when another option appears without id-filter value
+        const idReg = /^-/;
+        if (argv["id-filter"] === "true" || argv["id-filter"].match(idReg)) {
+            return finish(errHndl.getErrMsg("EMPTY_VALUE", "id-filter"));
+        }
+        options.id = argv["id-filter"];
         deviceLib.processResource(options, finish);
```

Comment:
Is this needed a comment for consistence?

Reply:
I removed.

Reply:
I removed it.